### ISSUE TITLE
the 8 parameters of unityeyes for generating training data

### ISSUE THREAD

**Issue by BlueWinters** [swook/GazeML#21]
For UnityEyes , what are 8 paramters(camera 4 & eye 4)? And is it important for the training?

**Comment by swook** [comment]
I generated UnityEyes images using the default parameters at 640x480 resolution with the setting "Fantastic". Due to a bug in rendering the Unity window via Wine on Linux I am not 100% sure which parameters the defaults were, but judging by the screenshot at https://www.cl.cam.ac.uk/research/rainbow/projects/unityeyes/tutorial.html it seems to be:

Camera: [0, 0, 20, 40]
Eyes: [0, 0, 30, 30]

As for whether they matter... this depends. For the purpose of training a gaze CNN to directly apply to MPIIGaze data, it was shown multiple times that improved performance can be achieved by sampling synthetic images to match the test data gaze distribution. See [Fig. 12 here](https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=7410785) and [Fig. 13 here](https://arxiv.org/abs/1711.09017). Having said that, it is not usually assumed that one has knowledge of test set labels at training time, and thus papers do not perform main evaluations in this manner.

For an unsupervised method of aligning gaze distributions between synthetic and real-world data, see [this ICLR paper from Kangwook Lee et al.](https://openreview.net/forum?id=SkHDoG-Cb).
